lib/StatusIndicator/styles.scss

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+
+$size-status-indicator: 1em !default;
+$cursor-status-indicator--hover: pointer !default;
+$color-status-indicator-label--text: rgb(111, 121, 129) !default;
+
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+
+.status-indicator__circle {
+  display: inline-block;
+  vertical-align: middle;
+  border-radius: 50%;
+  width: $size-status-indicator;
+  height: $size-status-indicator;
+  margin-right: .5rem;
+  margin-left: .25rem;
+}
+
+.status-indicator {
+  display: inline-block;
+  vertical-align: middle;
+  color: $color-status-indicator-label--text;
+}
+
+.status-indicator__label {
+  margin-right: .25rem;
+}
+
+.status-indicator__children .dateTime {
```

Comment:
I recommend we keep this flat and also inline with BEM so something like `status-indicator__date` would be good.

Failing that if you're trying to make the date look like a link (underline etc...) then I'd recommend using an anchor tag in the child markup and remove this styling all together.